components/widgets/inputs/users_emails_input.scss

```diff
@@ -203,3 +225,23 @@
         margin-left: 8px;
     }
 }
+
+.InputErrorBox {
+    margin-top: 2px;
+    width: 100%;
+    height: 60px;
+    background: rgba(197, 67, 72, 0.1);
+    border-radius: 4px;
+    display: flex;
+    align-items: center;
+
+    span {
+        height: 20px;
+        position: relative;
+        font-size: 16px;
+        line-height: 20px;
+        margin: auto;
+        text-align: center;
+        color: #FD5960
```

Comment:
Same with this colour.

Reply:
Fixed all instances of these to point to a variable `var(--error-text)` since it already existed.